src/main/kotlin/racingcar/domain/car/CarFactory.kt

```diff
@@ -0,0 +1,9 @@
+package racingcar.domain.car
+
+object CarFactory {
+    fun createCars(total: Int): List<Car> {
+        return buildList {
+            repeat(total) { add(Car()) }
+        }
+    }
```

Comment:
buildList는 처음 보네요! 👍

```suggestion
        fun createCars(total: Int): List<Car> = List(size = total) { Car() }
```

Fake Constructors를 활용해 볼 수도 있을 거 같네요.
다만 Fake Constructors 사용한다면 음수 처리가 Factory에 들어가야겠네요.

Reply:
아 이런 방법도 있군요! 적용해보도록 하겠습니다!